aoscp.mk

```diff
@@ -25,8 +25,8 @@ $(call inherit-product, device/htc/m8/device.mk)
 $(call inherit-product, vendor/aoscp/configs/common_full_phone.mk)
 
 # Device identifier. This must come after all inclusions.
-PRODUCT_NAME := aoscp_m8
-PRODUCT_DEVICE := m8
-PRODUCT_BRAND := htc
-PRODUCT_MANUFACTURER := htc
+PRODUCT_NAME := AOSCP_M8
```

Comment:
Roomservice won't be able to read the device when the variable is in all caps. Besides, this should be submitted to gerrit. We've stated all changes go through gerrit